eps/template.md

```diff
@@ -0,0 +1,52 @@
+# EP-\<issue number\>: Enhancement Proposal Template
+
+* Issue: \<link to issue\>
+* Status: Provisional|Prototyping|Implementable|Completed|Deferred|Rejected|Withdrawn|Replaced
+
+(See status definitions [here](README.md#status).)
+
+## TLDR
+
+(1-2 sentence summary of the proposal)
+
+## Goals
+
+(Primary goals of this proposal.)
+
+## Non-Goals
+
+(What is out of scope for this proposal.)
+
+## Introduction
+
+(Can link to external doc -- but we should bias towards copying the content into the EP as online documents are easier
+to lose -- e.g. owner messes up the permissions, accidental deletion)
+
+## API and Customer Driven Interfaces
+
+(... details, can point to PR with changes)
+
+## Use Cases
+
+(Describe the use cases for this feature)
+
+## User Experience and User Interface
```

Comment:
how is user interface different from customer driven interfaces?

Reply:
I'll collapse the two sections